i18n/de-formal.json

```diff
@@ -0,0 +1,80 @@
+	"ce_com_edit_not_exists"            : "Sie versuchten einen nicht existierenden Artikel zu editieren. Bitte überprüfen Sie dies erneut.",
+	"ce_nothing_deleted"                : "Kein Kommentar gelöscht.",
+	"ce_comment_delete_reason"          : "Kommentar wurde per SemanticComments GUI gelöscht.",
+	"ce_comment_deletion_successful"    : "Kommentar wurde erfolgreich gelöscht.",
+	"ce_comment_massdeletion_successful": "Kommentare wurden erfolgreich gelöscht.",
+	"ce_comment_deletion_error"         : "Kommentar konnte nicht gelöscht werden.",
+	"ce_comment_has_deleted"            : "löschte diesen Kommentar am",
+	"ce_com_default_header"         : "Erstellen",
+	"ce_com_ext_header"             : "Kommentare",
+	"ce_invalid"                    : "Sie haben keinen gültigen Kommentar eingegeben.",
+	"ce_reload"                     : " Seite wird neu geladen ...",
+	"ce_deleting"                   : "Lösche Kommentar ...",
+	"ce_full_deleting"              : "Lösche Kommentare vollständig ...",
+	"ce_delete"                     : "Möchten Sie diesen Kommentar wirklich löschen?",
+	"ce_delete_button"              : "Lösche Kommentar",
+	"ce_cancel_button"              : "Abbrechen",
+	"ce_full_delete"                : "Lösche diesen Kommentar komplett und entferne auch alle nachfolgenden Antworten zu diesem Kommentar.",
+	"ce_close_button"               : "Fenster schließen und Seite neu laden",
+	"ce_delete_title"               : "Klicken Sie hier um diesen Kommentar zu löschen",
+	"ce_edit_title"                 : "Klicken Sie hier um diesen Kommentar zu editieren",
+	"ce_edit_cancel_title"          : "Klicken Sie hier um den Editiermodus zu verlassen.",
+	"ce_reply_title"                : "Klicken Sie hier um auf diesen Kommentar zu antworten",
+	"ce_com_reply"                  : "Antworten",
+	"ce_edit_rating_text"           : "Bewerten Sie die Qualität dieses Artikels ",
+	"ce_edit_rating_text2"          : "(optional)",
+	"ce_edit_button"                : "Aktualisiere Kommentar",
+	"ce_com_show"                   : "Zeige Kommentare",
+	"ce_com_hide"                   : "Verstecke Kommentare",
+	"ce_com_view"                   : "Ansicht",
+	"ce_com_view_flat"              : "Flach",
+	"ce_com_view_threaded"          : "Eingerückt",
+	"ce_com_file_toggle"            : "Zeige Anhänge",
+	"ce_com_rating_text_short"      : "Durchschn. Artikelqualität",
+	"ce_com_rating_text"            : "Die durchschnittliche Qualität dieses Artikels basierend auf",
+	"ce_com_rating_text2"           : "Bewertung(en).",
+	"ce_com_toggle_tooltip"         : "Sie können die Kommentare durch Klicken öffnen/schließen",
+	"ce_form_toggle_tooltip"        : "Sie können das Kommentarformular durch Klicken öffnen/schließen.",
+	"ce_form_toggle_no_edit_tooltip": "Sie dürfen leider keine neuen Kommentare erstellen.",
+	"ce_edit_intro"                 : "Dieser Kommentare wurde zuletzt bearbeitet von",
+	"ce_edit_date_intro"            : "am"
+}

```

Comment:
"de-formal" ist keine vollständige Lokalisierung, d.h. in der Datei sollten nur Systemnachrichten enthalten sein, die nicht mit "de" identisch sind.